### DISCUSSION TITLE
Issue on displaying stats

### DISCUSSION THREAD

**Discussion by JoaoCioffi** [DenverCoder1/github-readme-streak-stats#548] [Q&A]
Hi everyone. 
I've just noticed that my stats were not displaying anymore. I tried to recompile a new setup using the demo lab and now it looks like this:

``` md
[![GitHub Streak](https://streak-stats.demolab.com?user=JoaoCioffi&theme=dark&hide_border=true&fire=25EB00&background=45%2C000000%2C0C167E&ring=25EB00&currStreakLabel=25EB00&sideLabels=25EB00&currStreakNum=FFFFFF&sideNums=FFFFFF)](https://git.io/streak-stats)
```

Are you guys having the same problem?

**Comment by DenverCoder1** [comment]
The connection to the Heroku server has been a bit slow today. You could always switch to hosting your own copy if you want, but this issue should go away on its own next time GitHub fetches the image.
